docs/_build/html/_sources/tutorial.rst.txt

```diff
@@ -0,0 +1,198 @@
+####################
+ CytoSnake Tutorial
+####################
+
+*******
+ About
+*******
+
+This tutorial assumes that you have follows the installation steps and
+you are ready to start taking off with CytoSnake!
+
+Cytosnake is a command line interface (CLI) tool that contains a
+multitude of workflows for analyzing morphological features obtained
+from microscopy images of cells.
+
+**********
+ Concepts
+**********
+
+.. image:: images/Cytopipe-cli_0.drawio.png
+
+Modes
+=====
+
+Modes provide options on how the user can change the functionality of
+CytoSnake. For example, if you would like to initialize your files for a
+specific workflow, you can simply type:
+
+using the `init` mode
+
+.. code::
+
+   cytosnake init <DATAFILES> <METADATA> <DATATYPE>
+
+-  `DATAFILE` will refer to the raw data that you are going to analyze,
```

Comment:
```suggestion
-  `DATAFILES` will refer to the raw data that you are going to analyze,
```

To match the command

Reply:
I think it would really help to have a concrete example of these placeholders (in addition to having the placeholders, they are helpful too!)

Reply:
Ah! I see that you have this below. You can reference this (e.g. "See below for a full working example" or something like this)